src/optimize-streams.ts

```diff
@@ -143,8 +142,8 @@ export class JSMinifyTransform extends GenericOptimizeTransform {
         babelTransform(content, {
           presets: [minifyPreset(null, {simplifyComparisons: false})],
           plugins: [
-            babelPluginSyntaxObjectRestSpread,
-            babelPluginSyntaxDynamicImport,
+            objectRestSpreadSyntax,
```

Comment:
Add todo/issue to combine with JSCompileTransform

Reply:
Just went ahead and did it.